bin/pyspark

```diff
@@ -82,4 +82,4 @@ fi
 
 export PYSPARK_DRIVER_PYTHON
 export PYSPARK_DRIVER_PYTHON_OPTS
-exec "$SPARK_HOME"/bin/spark-submit pyspark-shell-main "$@"
+exec "$SPARK_HOME"/bin/spark-submit pyspark-shell-main --name "PySparkShell" "$@"
```

Comment:
`--name MyName` works. So the final command will be like this:

`spark-submit pyspark-shell-main --name "PySparkShell" --name "MyName <other args>`

Then, MyName takes effect since it comes later than PySparkShell.